_commands/tools/nslookup.md

```diff
@@ -2,7 +2,9 @@
 ---
 
 nslookup
-------- 
+-------
+-------
+nslookup which stands for "name server lookup" is used to query internet name servers interactive
```

Comment:
This isn't grammatically correct. While you're fixing, I'd prefer if you'd include something more accessible to the course as I suggested in my previous review... Most people probably don't know what a name server is.